src/org/labkey/test/pages/core/admin/LimitActiveUserPage.java

```diff
@@ -0,0 +1,115 @@
+package org.labkey.test.pages.core.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class LimitActiveUserPage extends LabKeyPage<LimitActiveUserPage.ElementCache>
+{
+    public LimitActiveUserPage(WebDriver driver)
+    {
+        super(driver);
+        waitForPage();
+    }
+
+    public static LimitActiveUserPage beginAt(WebDriverWrapper wrapper)
+    {
+        wrapper.beginAt(WebTestHelper.buildURL("user", "limitActiveUsers"));
+        return new LimitActiveUserPage(wrapper.getDriver());
+    }
+
+    public LimitActiveUserPage userWarning(String value)
+    {
+        elementCache().userWarning.selectByVisibleText(value);
+        return this;
+    }
+
+    public LimitActiveUserPage limitActiveUsers(String value)
+    {
+        elementCache().limitActiveUsers.selectByVisibleText(value);
+        return this;
+    }
+
+    public String getUserWarningLevel()
+    {
+        return elementCache().userWarningLevel.get();
+    }
+
+    public LimitActiveUserPage setUserWarningLevel(String value)
+    {
+        elementCache().userWarningLevel.set(value);
+        return this;
+    }
+
+    public String getUserLimitLevel()
+    {
+        return elementCache().userLimitLevel.get();
+    }
+
+    public LimitActiveUserPage setUserLimitLevel(String value)
+    {
+        elementCache().userLimitLevel.set(value);
+        return this;
+    }
+
+    public String getUserWarningMessage()
+    {
+        return elementCache().userWarningMessage.getText();
+    }
+
+    public LimitActiveUserPage setUserWarningMessage(String value)
+    {
+        elementCache().userWarningMessage.clear();
+        elementCache().userWarningMessage.sendKeys(value);
+        return this;
+    }
+
+    public String getUserLimitMessage()
+    {
+        return elementCache().userLimitMessage.getText();
+    }
+
+    public LimitActiveUserPage setUserLimitMessage(String value)
+    {
+        elementCache().userLimitMessage.clear();
+        elementCache().userLimitMessage.sendKeys(value);
+        return this;
+    }
+
+    public ShowAdminPage save()
+    {
+        clickAndWait(elementCache().saveBtn);
+        return new ShowAdminPage(getDriver());
+    }
+
+    public ShowAdminPage cancel()
+    {
+        clickAndWait(elementCache().cancelBtn);
+        return new ShowAdminPage(getDriver());
+    }
+
```

Comment:
It doesn't look like you have any getters for the table at the bottom of the page that shows the current count of active users, remaining users and level values:
<img width="238" alt="Screen Shot 2022-06-08 at 10 18 03 PM" src="https://user-images.githubusercontent.com/12738200/172770038-fa496fd8-725b-42d3-b97d-4453a3b85304.png">

Reply:
Writing getter methods seemed not necessary, since the values(activeUser) is DB specific. 
Validating all the values are correctly populated in the warning message.